src/Administration/Resources/app/administration/src/app/snippet/en-GB.json

```diff
@@ -754,7 +754,9 @@
     },
     "sw-property-search": {
         "placeholderSearch": "Search properties...",
-        "selected": "{count} option selected | {count} options selected"
+        "selected": "{count} option selected | {count} options selected",
+        "noPropertiesFound": "No properties found",
+        "noPropertiesFoundDescription": "Try another search phrase"
```

Comment:
This one does not fit to the german one now.

Reply:
Changed it.